### PR TITLE
Add start page with hero

### PR DESCRIPTION
**Pull request by jodeldahl**
Added startpage with a simple hero section.

# Verify instructions
1. Open index.html
2. Verify that hero section exists with header, paragraph and image.
3. Verify that <title> of page is "Freaky Fashion".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (jodeldahl) [inline index.html:6]: Hoppsan!! Jag glömde visst ändra titel ;-)

Comment (CallePersson) [reply index.html:6]: Stämmer!

Comment (CallePersson) [inline index.html:6]: Title should be Freaky Fashion!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+    <div class="hero">
+        <h1>Freaky fabulous!</h1>
+        <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem, labore asperiores quibusdam vero odit, suscipit totam magnam, fugiat exercitationem placeat tempora nihil esse. Praesentium enim placeat voluptatem tenetur, explicabo corporis.</p>
+    </div>
```
Comment (CallePersson) [inline index.html:12]: Hero should have an image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="hero">
+        <h1>Freaky fabulous!</h1>
```
Comment (CallePersson) [inline index.html:10]: heading should be h2

**Review by CallePersson — CHANGES_REQUESTED** [review]

**Pull request closed without merging**